plugin/ruby.vim

```diff
@@ -56,8 +59,12 @@ module VIM
 end
 
 class RubyTest
+  def initialize
```

Comment:
Maybe it would make more sense to take the variable in the `initialize` instead of in each method? Then, maybe you can consolidate all of the `RubyTest.new` into a single place and pass in the vim variable. We could probably do this with the rest of the variables as well.